Cubical/Algebra/CommSemiring/Base.agda

```diff
@@ -14,6 +14,8 @@ private
 record IsCommSemiring {R : Type ℓ}
                   (0r 1r : R) (_+_ _·_ : R → R → R) : Type ℓ where
 
+  constructor iscommsemiring
```

Comment:
Are you using that?
We say in CONTRIBUTING.md, that copattern matching should be the preferred way of defining elements of record types and if you follow that, you'll never need those constructors - so it should actually be good to *not* define those constructors, to not guide people in the wrong direction.

Reply:
I think I was just copy-pasting some other algebra file for a more convenient constructor for IsCommSemiring, but one was already pushed so this doesn't matter anymore.